Cargo.toml

```diff
@@ -14,7 +15,7 @@ exclude = [
 
 [dependencies]
 errno = "0.2"
-error-chain = {version = "0.12", default-features = false}
+failure = "0.1"
```

Comment:
While I acknowledge that `error_chain` is dead, I'd rather not do this for two reasons:
 * `failure` is dead too
 * this introduces a dependency on a C library via `backtrace`

For the future, I'm keeping an eye on `snafu` but I'd like to see this fixed first: https://github.com/shepmaster/snafu/issues/113.

Reply:
Interesting. The problem with error_chain I encountered (if I remember correctly) was that the errors where not Send.

Reply:
Indeed, that's painful if your consumer is on `failure` as different decisions were taken between the two libraries. You'll need to re-wrap the error at the border, for the moment.